aries-site/src/layouts/content/AccessibilitySection.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable react/prop-types */
+import React, { useState, useMemo, useEffect } from 'react';
+import { WCAGRuleDetail, WCAGRuleSummary } from '.';
+import componentData from '../../data/wcag/components.json';
+
+export const AccessibilitySection = ({ title }) => {
+  const [data, setData] = useState();
+
+  const componentInfo = useMemo(() => {
+    if (!title || !componentData) {
+      return [];
+    }
+
+    const component = componentData.find(item => item[title.toLowerCase()]);
+    return component ? component[title.toLowerCase()] : [];
+  }, [title]);
+
+  useEffect(() => {
+    fetch(
+      'https://raw.githubusercontent.com/w3c/wcag/refs/heads/main/guidelines/wcag.json',
+    )
+      .then(response => {
+        if (!response.ok) {
+          throw new Error('Error fetching data');
+        }
+        return response.json();
+      })
+      .then(fetchedData => {
+        setData(fetchedData);
+      })
+      .catch(error => {
+        console.error('Error:', error);
+      });
+  }, []);
+
+  // create a map of each of the successCriteria ->
+  // num to make it way easier and faster over time.
+  // This will be used to
+  // look up the success criteria by rule number
+  const successCriteriaMap = useMemo(() => {
+    if (!data) return new Map();
+
+    const map = new Map();
+    data.principles?.forEach(principle => {
+      principle.guidelines?.forEach(guideline => {
+        guideline.successcriteria?.forEach(criterion => {
+          map.set(criterion.num, criterion);
+        });
+      });
+    });
+    return map;
+  }, [data]);
+
+  // Compare the component info with the success criteria
+  // and return the status of each rule.
+  const comparisons = useMemo(() => {
+    return componentInfo.map(rule => {
+      const ruleNum = rule.rule;
+      const successCriterion = successCriteriaMap.get(ruleNum);
+
+      if (successCriterion) {
+        const extractedData = {
+          id: successCriterion.id.split(':')[1],
+          num: successCriterion.num,
+          // WCAG 2.2 is the latest version, so grab that if available
+          version: (() => {
+            if (successCriterion.versions) {
+              return successCriterion.versions.includes('2.2')
+                ? '2.2'
```

Comment:
Thanks. Makes sense. Rather than hardcode it here though I think it should be in the component info (and probably per component.) When we decide to update to WCAG 2.3 or whatever, would we do all components at once or have a period of time where some components were still 2.2?

Reply:
good point will do

Reply:
added the `version` prop for us to be able to control in the future if we do updates